src/framework/app-base.js

```diff
@@ -20,6 +20,8 @@ import { http } from '../net/http.js';
 import {
     PRIMITIVE_TRIANGLES, PRIMITIVE_TRIFAN, PRIMITIVE_TRISTRIP
 } from '../graphics/constants.js';
+import { setProgramLibrary } from '../graphics/get-program-library.js';
```

Comment:
LOL, I know this is intentional, but damn, it looks like a bug.

Reply:
maybe I rename it to 'program-library-access.js'?